lib/js/dom.nim

```diff
@@ -1183,9 +1183,10 @@ proc contains*(c: ClassList, class: cstring): bool
 proc toggle*(c: ClassList, class: cstring)
 
 # Style "methods"
-proc getAttribute*(s: Style, attr: cstring, caseSensitive=false): cstring
```

Comment:
Because it has wrong name, https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty
should be getPropertyValue, removeProperty, setProperty instead of getAttribute, removeAttribute and setAttribute.